core/templates/domain/skill/SkillMasteryObjectFactorySpec.ts

```diff
@@ -0,0 +1,62 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Tests for SkillMasteryObjectFactory.
+ */
+
+import { SkillMasteryObjectFactory } from
+  'domain/skill/SkillMasteryObjectFactory';
+
+describe('Skill mastery object factory', () => {
+  let skillMasteryObjectFactory: SkillMasteryObjectFactory;
+
+  let skillMasteryDictSample = {
```

Comment:
I actually suggest inlining this within the tests. The values defined here are directly relevant to verifying that the tests are correct, and the bodies of tests should include all of the details necessary to verify test correctness.

Reply:
Done.